_maps/shuttles/whiteship_pubby.dmm

```diff
@@ -40,7 +40,7 @@
 /turf/open/floor/iron/dark,
 /area/shuttle/abandoned/crew)
 "ah" = (
-/turf/closed/wall/mineral/titanium/overspace,
+/turf/closed/wall/mineral/titanium/,
```

Comment:
bad path

Reply:
also these keys are likely to not exist since titanium wall and an area is likely to be already defined

Reply:
ah shit my bad